routes.js

```diff
@@ -39,6 +39,9 @@ module.exports = function (app, router) {
     //route for getting pokemon details by specific name
     router.get('/pokemon/name/:name', pokemon.getByName);
 
+    //route for getting pokemon details by description
+    router.get('/pokemon/description/:description', pokemon.getByDescription());
```

Comment:
Unfortunately i cannot test the API any more as it seems the develop was pushed to docker now (and not this branch with the description feature) but i think it should be fine like this!